### ISSUE TITLE
Links in `.hero.is-light` are not discernable from regular text

### ISSUE THREAD

**Issue by Vinnl** [jgthms/bulma#1646]
This is about **Bulma**.

### Overview of the problem

This is about the Bulma **CSS framework**.

I'm using Bulma **version** 0.6.1
My **browser** is: Firefox 58, Chromium 64 and Epiphany 3.18.11.

I am sure this issue is **not a duplicate**

### Description

With a layout like this:

```
<section class="hero is-light">
    <p class="content">
        This is content with <a href="#" title="Some title">a link[/a> in it.
    </p>
</section>
```

The `<a>` is not discernible from the rest of the content, whereas in a plain `.hero`, for example, it at least has a different colour.

### Steps to Reproduce

Use the code above.

### Expected behavior

Links are discernible from the rest of the text (preferably in more than one modality, but at least through a different colour).

### Actual behavior

The link is styled the same as the rest of the text.

**Comment by Vinnl** [comment]
PR #2212 still requires some work, apparently, so this issue is not fixed yet. That said, there is also #2196, which appears to be a duplicate.

**Comment by Vinnl** [comment]
Nope, not fixed yet.

**Comment by Vinnl** [comment]
Still an issue with current Bulma.

**Comment by Vinnl** [comment]
Appears to be fixed in Bulma 0.9.
